### PR TITLE
Creates trigger and test for form 20 10206

### PR DESCRIPTION
**Pull request by MarchandMD**
## Summary

Creates trigger and test for form 20 10206

## Related issue(s)

[Link to related ticket - https://app.zenhub.com/workspaces/forms-strike-team-620c2914e703390013c4b414/issues/gh/department-of-veterans-affairs/vanotify-team/1213](https://app.zenhub.com/workspaces/forms-strike-team-620c2914e703390013c4b414/issues/gh/department-of-veterans-affairs/vanotify-team/1213)

## Testing done

- [x] *New code is covered by unit tests*

- *Describe the steps required to verify your changes are working as expected. Exclusively stating 'Specs run' is NOT acceptable as appropriate testing*

```bash
bundle exec rspec modules/simple_forms_api/spec/services/confirmation_email_spec.rb:207
```

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by nathanbwright — DISMISSED** [review]

File: modules/simple_forms_api/spec/services/confirmation_email_spec.rb [hunk 1]
```diff
@@ -203,4 +203,37 @@
...
+
+  describe '20_10206' do
+    let(:data) do
+      fixture_path = Rails.root.join(
+        'modules', 'simple_forms_api', 'spec', 'fixtures', 'form_json', 'vba_20_10206.json'
+      )
+      JSON.parse(fixture_path.read)
+    end
+
+    describe 'citizen or non-citizen' do
```
Comment (Thrillberg) [inline modules/simple_forms_api/spec/services/confirmation_email_spec.rb:215]: This is totally a nit but do we need this `describe` block? I don't see any other `describe` blocks that present an alternative path so this should work with the `it` block below coming directly after we define `:data`, right?

**Review by Thrillberg — APPROVED** [review]
Love it! Thanks! One tiny nit below, feel free to ignore.

File: modules/simple_forms_api/spec/services/confirmation_email_spec.rb (hunk 1 above)
Comment (MarchandMD) [reply modules/simple_forms_api/spec/services/confirmation_email_spec.rb]: Yes! I had separate blocks for citizen/non-citizen; but when those were deemed unnecessary, I didn't remove the `describe`

**Review by Thrillberg — APPROVED** [review]
🎉

**Pull request merged**